src/stacks-hierarchy/Root_TabNavigatorStack/TabNav_TicketingStack/FareContracts/Component/TransportMode.tsx

```diff
@@ -29,24 +60,42 @@ const TransportMode = ({
           disabled={disabled}
         />
       ))}
-      <ThemeText type="label__uppercase" color={'secondary'}>
-        {modes
-          .map((tm) => t(FareContractTexts.transportMode(tm.mode)))
-          .join('/')}
+      {modesCount > modesDisplayLimit && (
+        <View style={[styles.multipleModes, boxHeight]}>
+          <ThemeText color={'transport_other'} type="label__uppercase">
+            +{modesCount - modesDisplayLimit}
+          </ThemeText>
+        </View>
+      )}
+      <ThemeText
+        type="label__uppercase"
+        color={'secondary'}
+        accessibilityLabel={t(
+          FareContractTexts.transportModes.a11yLabel(
```

Comment:
Is the idea of the `accessibilityLabel` to describe all the modes? Because it is only talking the first two.